makelove/macos.py

```diff
@@ -0,0 +1,245 @@
+import io
+import os
+import shutil
+import struct
+import sys
+from datetime import datetime
+from zipfile import ZipFile
+from urllib.request import urlopen, urlretrieve, URLError
+
+from PIL import Image
+
+from .util import eprint, tmpfile
+from .windows import get_default_love_binary_dir, get_download_url
+
+
+def download_love(version, platform):
+    """
+    Note, mac builds are stored as zip files because extracting them
+    would lose data about symlinks when building on windows
+    """
+    target_path = get_default_love_binary_dir(version, platform)
+    print("Downloading love binaries to: '{}'".format(target_path))
+
+    os.makedirs(target_path, exist_ok=True)
+    try:
+        download_url = get_download_url(version, platform)
+        print("Downloading '{}'..".format(download_url))
+        with urlopen(download_url) as response:
```

Comment:
I messed this up for windows too, but I think you can use urlretrieve here instead.